### PR TITLE
Improve Comparable take zelf as reference

### PR DESCRIPTION
**Pull request by qingshi163**

### CODE REVIEW

File: vm/src/types/slot.rs [hunk 1]
```diff
@@ -992,58 +992,35 @@ pub trait Comparable: PyPayload {
         vm: &VirtualMachine,
     ) -> PyResult<PyComparisonValue>;
 
+    #[inline]
```
Comment (fanninpm) [inline vm/src/types/slot.rs:995]: This seems unrelated.

Comment (youknowone) [reply vm/src/types/slot.rs:995]: other functions anyway have it

**Review by youknowone — APPROVED** [review]
